Resources/Private/Templates/DataSheetImport/Index.html

```diff
@@ -0,0 +1,74 @@
+    <h1><f:translate key="module_name" extensionName="xlsimport" /></h1>
+    <f:flashMessages queueIdentifier="core.template.flashMessages" />
+    <f:be.infobox title="{f:translate(key:'importinfo', extensionName: 'xlsimport')}" state="-1">
+        <f:translate key="importinfo_format" extensionName="xlsimport" />
+        <ul>
+            <li>Excel 2007 (.xlsx)</li>
+            <li>Excel5 (.xls)</li>
+            <li>Excel2003XML (.xml)</li>
+            <li>OpenDocument Calc (.ods)</li>
+            <li>Gnumeric (.gnumeric)</li>
+            <li>CSV (.csv/.txt)</li>
+        </ul>
+    </f:be.infobox>
+    <form
+        action="{f:be.uri(route: 'web_xlsimport', parameters: '{action: \'upload\', id: \'{page}\'}')}"
+        enctype="multipart/form-data"
+        method="post"
+    >
+        <div class="typo3-TCEforms">
+            <fieldset class="form-section">
+                <div class="row">
+                    <div class="form-group col col-sm-6">
+                        <label class="t3js-formengine-label" for="table">
+                            <f:translate key="tableselect" extensionName="xlsimport" />
+                        </label>
+                        <select
+                            class="form-select"
+                            name="table"
+                            id="table"
+                        >
+                            <f:for each="{allowedTables}" as="tableLabel" key="allowedTable">
+                                <option value="{allowedTable}">{tableLabel}</option>
+                            </f:for>
+                        </select>
+                    </div>
+                </div>
+                <div class="row">
+                    <div class="form-group col col-sm-12">
+                        <label class="t3js-formengine-label" for="fileUpload">
+                            <f:translate key="fileselect" extensionName="xlsimport" />
+                        </label>
+                        <input type="file" class="btn btn-default" name="file" id="fileUpload">
+                    </div>
+                </div>
+                <div class="row">
+                    <div class="form-group col col-sm-12">
+                        <input type="checkbox" name="encoding" id="encoding" value="1">
+                        <label class="t3js-formengine-label" for="encoding"><f:translate key="encoding" extensionName="xlsimport" /></label><br>
+                        <input type="checkbox" name="deleteRecords" id="deleteRecords" value="1">
+                        <label class="t3js-formengine-label" for="deleteRecords"><f:translate key="deleteRecord" extensionName="xlsimport" /></label>
+                    </div>
+                </div>
+                <div class="row">
+                    <div class="form-group col col-sm-12">
+                        <button type="submit" class="btn btn-default">{f:translate(key:'upload', extensionName: 'xlsimport')}</button>
```

Comment:
No need for using inline notation here. I would suggest to use the tag variant here.
Being consequent and predictable is a good thing. Inline notation should only be used, when used to put things in tags or tag attributes. ANd even than, I personally would eager put it in a variable first and use the "variable" than.